app/src/main/java/com/keylesspalace/tusky/util/SpanUtils.kt

```diff
@@ -0,0 +1,133 @@
+package com.keylesspalace.tusky.util
+
+import android.text.Spannable
+import android.text.Spanned
+import android.text.style.CharacterStyle
+import android.text.style.ForegroundColorSpan
+import android.text.style.URLSpan
+import android.util.Patterns.WEB_URL
+import java.util.regex.Pattern
+
+class SpanUtils {
```

Comment:
This is Kotlin, not Java, no need for a class when everything is static anyway. You can remove class/companion object and @JvmStatic, reference the functions in Java as `SpanUtilsKt` afterwards.